app/main.py

```diff
@@ -6,9 +6,11 @@
 
 app = FastAPI()
 
+security_token = "qwertyuiopasdfghjklzxcvbnm" 
 
 @app.get("/")
 def root():
+    print("I am here")
     return {"message": "Fast API in Python"}
```

Comment:
Hardcoding a security token within the application's source code is a significant security risk. This exposes the token to anyone who has access to the source code, increasing the risk of unauthorized access. It's recommended to use environment variables or a secure vault service for storing sensitive information like security tokens.

**Label**: security

**Code suggestion**

```python
import os

security_token = os.getenv('SECURITY_TOKEN')
```